apps/rehash.c

```diff
@@ -198,7 +194,7 @@ static int handle_symlink(const char *filename, const char *fullpath)
  */
 static int do_file(const char *filename, const char *fullpath, enum Hash h)
 {
-    STACK_OF (X509_INFO) *inf = NULL;
+    STACK_OF (X509_INFO) * inf = NULL;
```

Comment:
This should be `STACK_OF(X509_INFO) *inf = NULL`

The reason `util/openssl-format-source` didn't fix this is the space between `STACK_OF` and `(X509_INFO)`...  it shouldn't have been there.